packages/core/src/render3/instructions/instructions.ts

```diff
@@ -1093,11 +1095,30 @@ export function elementEnd(): void {
 
 
 /**
- * Flushes all the lifecycle hooks for directives up until (and excluding) that node index
+ * Selects an index of an item to act on and flushes lifecycle hooks up to this point
```

Comment:
```suggestion
 * Selects an element for later binding instructions.
```

Look at it from developer point of view. The fact that we use it to flush lifecycle hooks is an implementation details so we should not be leaking it.